src/pages/business-adopters/index.tsx

```diff
@@ -0,0 +1,104 @@
wvD8DfP4mncqWaj/hFalwMSbnQCTcDloV7GO5M4B4G7BuEm9GB6zQdKBZ3xidMmGBvaWm5DQmyFmNRLg7hTtyVSUlJa6qqqrxexCclOHHixJnAcV82m22hXl0o//9AWhyqsoAfjrHSuaAxDko6BnSOI74Ls7Adu3bt2utmQIhQm+DKz1ZBKPctJhBVappCkh0G4+5/ZsHDu+/LL7+sFnEojr5IRN5FoJENXk5D/fUId6empm7CJEaLHlA6ogpDNaIhn2LKJlaeQYyj0SdVINKvgPF/J4GqMFc4DjCKzjl48OAjEPZsWDGyWF7XyJEjb8UU91lkiFNDwnNOcWEZFkGwC6DEpWLh9vb2U8U0hFgHl/m1CItEHLyPB4+r0J7zVXqutjHMwhTIYA0UdRE62uMPLqAsObDKm9Qy4O9lxPPUNIVQghuAs0yFQSEeQvwJNU3hmDFjctDWdcBzu161ftRBfRFS30tiJZGIg8HvtHTS0tKSNbADSGuVxQMFjRsOWuv11n/QEVdBSK+ggFZZ3DRQfgASJei4G9zAzoh7scoF97tcoCnvN4k6LwTv28GDW1k0hfC1EGUOOvQzWLw0TV7YSYwb++ChfReE3MoiEkXdJAOyxkFfEVcYaL/TRPjiBDir8eTMwhM2DK4j0W63pwI2EbDncIvvA52Cp6FQpEXCQIN/J8JQ5hjujbj/hts5EFfz0XHP06Kcmo52SNYUdb6IesQOUcDXV4DhhX6Pawgs4WIPSAQSeJjmQUb0wLgv1N+Gm1y03/5xF9KJRFxhdOrwAsHNfAUX9XtyBRgntNEKMQaWXwA2F436mabAJDENYUyDMGhw7bxICFC8s1B2Ou4ZULzxyBCt3GlwXTNc6F0R3IRK3OMLxI+AJ+JvONzjKYjfJjKBttwNJaM1n4hdoKl99Xd5enp6Gnjon5mZmQqZXY/bPUQIpuKYKIzIID39ENjNuJfiprWRhWI+4jSucbsvxGmQLF6rMA7YowJIGSGM5WqaQjzx2jJidqTjOSJB8PIkPQwqDPG1gL2tphFSH4wT0mFHIaOhKhHUdRQTiEV4KNsJtm3btlYozpvAOaHiBBOGNPAJpgIjXAxIR6Axv8T+zSwwT2sduhfy0rCvQ/lOV4X0WBERT2yFmKY46JYDzw1G2qOMOyM6EQ/lBB8f61RDsOkCnMpEZCsFD10i6swUaO+P5BpSTCwMGnULGoQ/2lRuw+1WFnRsFe5S5MlCgz2iyE8VASjvMcugPMC002mPMmL5SMdRt3vh0kVbO24h/mrFenXKiNlBxVNSUjyMAGh7jOmCIqaD3OUKgxnElWjEa+AlxcUPPgPGi7A2MRSmMht3PtKGCqNtAyyM146yFqeL0z+YNlSM8ZUefx44XcxfWNV5aGNYlFyF0YGpGDN4kMLimlMBpk6dmnTgwIHnxUwoRwGUhNYaQrpQV8DKZVBBVB8aA4UxYMX84IgLCx34I7HZUIgODPSaCXb48OGzEQyhOF3I22agLL6eQFotFS+vGQaUVusW3CuJmGVpTfRgWDz3oFokHGJcXBagRTovd4h29xFpI+1+whD3eADAW1APNZYh3G111ZEs1hVuPOIKgwbmi0whvRNCcCoAOks7g/hMxBXivjrwSwGPojRg9Li0syLw4B4zQJlo0dB9IS8VY6rJbkCYEbR1t0gCbRan2M4s1OkBQ9o95gLvDWJ50NM7VpEk4ohxPIA0XRbHTYPEfDUOuiH1fUiFqFI0ciBmL6eoN4R+FmY+RYDPVJmiEB20Xk0jT1zMIhqj1DwKkeZYxX0aoa+nqlIsg/g9GBe5hYKV01MBmyviQDhuxcS0uxbpz8V8xH8H/s8QYXCfNsB+ijZtCvJcj8e+D2guIFes0saSPbX5J2qaQtpbUtOJiYniGhKBJ4v1g59JkM9DKr5eiPZ9LcBPQzs8jnEg/VvQ8LByAr7PqK+O8VkQT86O1tZWnzjI3DNo0KAlcElOPPjzj8R9JjA9nZQMDXwDitUPDbkdiFf6IooFqE04HfctcE4jPNA4HbyUgw6t9MpYOSXhDKA8ugCrQb1/6Ex1/gK2BOXWCbBcxHeCxofI+w55mdjLOg+hkw7M/ELkPybgG0bR4W+Bh6dQVnVFF4LWDtB+G7T7YDvgX1HYrUCIb8U6kntpoLKysgFyOAK4c9sDdPqi/i0o/xZgZ+K+BTB/bubvwDsPt/MC/hugSRONVsT/BXe2KyvowJfp9yAGhn2NKzxwKQHhNGMjcob2LAwY/18wfI1XAR8A2j6gFWEVBbxcj/gbatpXCEWcCYUVlUO1ZK+CD48n3YgO2tICHkagM4/Amg2A26gRcD/DCq5zw1OFwUreCyV2bw6qcG0IuifB3wRYPQ83Cxk9Dt4e1OIbpUHnIbiiJ9R8WNl+UNpq0AjYiqANAelCyC5JZU4vRAM2QBBjtcpCuHjaZyPYqleOYCj7MW7aizG80LgNwJmD2+37dZAbkX+3VlkID3AFliAP0YdxawfRhCJetCL6NPB91SXis5kzZz4HANE2NMHggdzGtVplIUJQzuXI1z1HDDiNwTxcLpURL9exjp8C1iTC1ThoHMLtU8YqrjYMSKuoEJ5qn08MGGgB2iHcH0DbtX6cSLgvaD7HUzQLZcj95LgydiL8Izr4T9hgvABP8Sy1AJa2f653XsX1tN8JvGzQovMw+FIq34V0FYT+AlkElYZRSOdjMJNZiLKXAmcw7iTE60GHnvpNWAh7sby83D2IpN1lnKd5RqUHvGq0d7GaFkO0YyjacTvojQXeGIS0SUr8fY4jny/QMr2IL8bpGClWwRehzNkoQ4PkGoT0vwlFcGEtkN9KwJwPPCz5BijeX8XyFId8hsHS3Q0atLE7FCBy5W/jLgIdspSLEHdeeAhJjtZlScCSgCUBSwKWBCwJWBKwJGBJwJKAJQFLApYELAlYErAkYEnAkoAlAUsClgQsCVgSsCRgSSBkCfwTBEXYlhQii0kAAAAASUVORK5CYII=",
```

Comment:
Can we refer to a picture from the repo or a URL?

Reply:
Yes, it's just a sample of different picture styles.